var/spack/repos/builtin/packages/beast2/package.py

```diff
@@ -0,0 +1,51 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Beast2(Package):
+    """BEAST is a cross-platform program for Bayesian inference using MCMC
+       of molecular sequences. It is entirely orientated towards rooted,
+       time-measured phylogenies inferred using strict or relaxed molecular
+       clock models. It can be used as a method of reconstructing phylogenies
+       but is also a framework for testing evolutionary hypotheses without
+       conditioning on a single tree topology."""
+
+    homepage = "http://beast2.org/"
+    url      = "https://github.com/CompEvol/beast2/releases/download/v2.4.6/BEAST.v2.4.6.Linux.tgz"
+
+    version('2.4.6', 'b446f4ab121df9b991f7bb7ec94c8217')
+
+    depends_on('jdk')
+
+    def setup_environment(self, spack_env, run_env):
+        run_env.set('BEAST', self.prefix)
+
+    def install(self, spec, prefix):
+        install_tree('bin', prefix.bin)
+        install_tree('examples', join_path(self.prefix, 'examples'))
+        install_tree('images', join_path(self.prefix, 'images'))
+        install_tree('lib', prefix.lib)
+        install_tree('templates', join_path(self.prefix, 'templates'))
```

Comment:
Now that #4591 has been merged, you can do `self.prefix.examples`, `self.prefix.images`, `self.prefix.templates`, etc.